src/test/java/org/datavyu/plugins/ffmpeg/MpvMediaPlayerTest.java

```diff
@@ -0,0 +1,47 @@
+package org.datavyu.plugins.ffmpeg;
+
+import javafx.application.Application;
+import javafx.stage.Stage;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import javax.swing.*;
+import java.net.URI;
+
+public class MpvMediaPlayerTest extends MediaPlayerTest {
+
+    private static class MpvJxApplication extends Application {
```

Comment:
MpvJxApplication can't be an inner class, JavaFX Application must be in a separate class and called from this class with Platform.runLater(() -> start(new Stage())) in order to make it work